wire-runtime/src/test/java/com/squareup/wire/RedactedTest.java

```diff
@@ -15,73 +15,16 @@
  */
 package com.squareup.wire;
 
-import com.squareup.wire.protos.redacted.NotRedacted;
 import com.squareup.wire.protos.redacted.Redacted;
-import com.squareup.wire.protos.redacted.RedactedChild;
-import com.squareup.wire.protos.redacted.RedactedCycleA;
-import com.squareup.wire.protos.redacted.RedactedRepeated;
-import com.squareup.wire.protos.redacted.RedactedRequired;
 import java.io.IOException;
-import java.util.Arrays;
-import java.util.Collections;
 import org.junit.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.Assert.fail;
 
 public class RedactedTest {
-
   @Test
   public void testRedacted() throws IOException {
     assertThat(new Redacted.Builder().a("a").b("b").c("c").build().toString())
         .isEqualTo("Redacted{a=██, b=b, c=c}");
   }
-
-  @Test
-  public void testRedactor() {
```

Comment:
Maybe it could go through a serialization/deserialization path.

I'm not worried about the specific API (since as you say this is v2) but I want to maintain the capability.